test_heterogeneous_sequences.py

```diff
@@ -0,0 +1,157 @@
+#!/usr/bin/env python3
+
+from unittest import TestCase
+from dark.reads import Read, Reads
+from collections import Counter, defaultdict
+from heterogeneous_sequences import heterogeneousSites, compareToRef
+
+
+class TestHeterogeneousSites(TestCase):
+    """
+    Test the heterogeneousSites function.
+    """
+
+    def testOneRead(self):
+        read = Read('id', 'ACCG')
+        reads = Reads([read])
+
+        self.assertEqual({}, heterogeneousSites(reads, len(read), 1))
+
+    def testHomogeneousReads(self):
+        read = Read('id', 'ACCG')
+        reads = Reads([read, Read('id2', 'ACCG')])
+
+        self.assertEqual({}, heterogeneousSites(reads, len(read), 1))
+
+    def testHeterogeneousReadsOneDifference(self):
+        read = Read('id', 'ACCG')
+        reads = Reads([read, Read('id2', 'ACCC')])
+
+        self.assertEqual({3: Counter({'G': 1, 'C': 1})}, heterogeneousSites(reads, len(read), 1))
+
+    def testHeterogeneousReadsTwoDifferences(self):
+        read = Read('id', 'ACCG')
+        reads = Reads([read, Read('id2', 'TCCC')])
+
+        self.assertEqual({0: Counter({'A': 1, 'T': 1}), 3: Counter({'G': 1, 'C': 1})}, heterogeneousSites(reads, len(read), 1))
```

Comment:
You don't need the `Counter` here. Just use a `dict`.

Reply:
Done